app/components/hero/hero.styl

```diff
@@ -116,39 +121,64 @@
             -moz-margin-start 25px
             padding 0 25px
             border-radius 50px
-            background-color #ff00aa
-            background-color var(--active-color)
+            background-color rgba(131, 74, 223, 0.85)
+            border 2px solid var(--white-color)
+            // background-color var(--active-color)
           .sign-in-btn:hover
             // transition color 0.5s linear
             background-color #fff
             background-color var(--white-color)
             color var(--active-color)
           .sign-in-btn:focus
-            background-color #ff00aa
-            background-color var(--active-color)
+            background-color rgba(131, 74, 223, 0.85)
+            // background-color var(--active-color)
             color var(--white-color)
       .header-content
-        padding-top 6%
+        // padding-top 6%
         display flex
-        justify-content center
+        justify-content space-around
         h1
           letter-spacing: 1px
-          font-size 3rem
+          font-size 72px
         .header-content-wrap
           z-index 1
-          width 30%
-          padding 0 2rem
+          width 40%
+          padding 15% 0 0 3%
+          .try-for-free-btn
+            text-transform uppercase
+            font-size: 24px;
+            display: flex;
+            width: 300px;
```

Comment:
Pls **never** set a fixed width to text elements
What if it will be another language? Or we'll need to change a text?